### DISCUSSION TITLE
过期：（逻辑重写）建议修改rife滤镜限制分辨率的逻辑

### DISCUSSION THREAD

**Discussion by Tansuikarin** [hooke007/mpv_PlayKit#133] [回收]
目前的逻辑是检测短边大于1080则限制到1080 但有很多压制版电影是3840*16xx这样21：9的分辨率 应该以长边检测限制才能正确缩到1080p

**Comment by hooke007** [comment]
1080p就是按照短边的定义来的
难道16xx不是大于1080

**Reply by Tansuikarin** [reply]
确实 是我表述不清了 应该是缩到平时常见的1920×810分辨率 现在这样缩下来是2560×1080

**Reply by hooke007** [reply]
我确实就是这么设计的，考虑了其他更杂比例的视频。
这个需求可以在rife前插一个预缩放滤镜轻松实现

针对被人为裁切的这类片源
https://github.com/hooke007/MPV_lazy/blob/dcd8a123c84351895881f1994d963098565f1b64/portable_config/input.conf#L60
它比预缩放更合适

**Reply by Tansuikarin** [reply]
原来如此 多谢解答
